schema.sql

```diff
@@ -12,4 +12,52 @@ CREATE TABLE animals (
     PRIMARY KEY (id)
 );
 
-ALTER TABLE animals ADD COLUMN species varchar(50) NOT NULL;
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN species varchar(50) NOT NULL;
+
+CREATE TABLE owners (
+    id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    full_name VARCHAR(100),
+    age INTEGER
+);
+
+CREATE TABLE species (
+    id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(50)
+);
+
+ALTER TABLE animals
+    DROP COLUMN species,
+    ADD COLUMN species_id INTEGER,
+    ADD COLUMN owner_id INTEGER,
+    MODIFY COLUMN id INTEGER NOT NULL AUTO_INCREMENT;
+
+ALTER TABLE animals
+    ADD CONSTRAINT fk_species_id FOREIGN KEY (species_id)
+        REFERENCES species(id),
+    ADD CONSTRAINT fk_owner_id FOREIGN KEY (owner_id)
+        REFERENCES owners(id);
+
+CREATE TABLE vets (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name TEXT,
```

Comment:
- [ Required ] I think it would be better if you made the `name` field type `varchar(255)`. The `TEXT` data type is usually used to represent multi-line texts such as descriptions or paragraphs of text. Since the name field would never be more than `255` characters the `TEXT` data type is not the best data type to use. Kindly rectify this.